examples/hybrid_jobs/8_Creating_Hybrid_Job_Scripts/Creating_your_first_Hybrid_Job.ipynb

```diff
@@ -526,39 +526,47 @@
    "source": [
```

Comment:
<p>...The script can be uploaded directly <strong>from your computer</strong>...</p><p> </p><p>...After the algorithm is specified, you <strong>have the option</strong> to skip directly to the review and create page...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>